snuba/web/db_query.py

```diff
@@ -34,11 +34,11 @@
     RateLimitExceeded,
 )
 from snuba.util import force_bytes, with_span
-from snuba.utils.codecs import JSONCodec, JSONData
+from snuba.utils.codecs import JSONCodec
 from snuba.utils.metrics.timer import Timer
 from snuba.web import QueryException, QueryResult
 
-cache: Cache[JSONData] = RedisCache(
+cache: Cache[Result] = RedisCache(
```

Comment:
I think this is not supposed to work.
I had already taken care of this file but never had time to merge 
https://github.com/getsentry/snuba/pull/1642/files